### PR TITLE
Make nukies usable

### PR DESCRIPTION
**Pull request by metalgearsloth**
- Spawn points that work
- Radar default range bumped up
- Used the infiltrator instead
- Spawning works

I playtested it and it was working okay so anything new pops up then I'm gonna screm. Gameticker spawning is weird as so I may have misunderstood the logic going on.

:cl:
- fix: Nukies are now playable.

### CODE REVIEW

File: Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs [hunk 1]
```diff
@@ -88,22 +91,22 @@ private void OnPlayersSpawning(RulePlayerSpawningEvent ev)
...
         for (int i = 1; i < aabbs.Length; i++)
         {
             aabb.Union(aabbs[i]);
         }
 
-        var (_, grid) = _mapLoader.LoadBlueprint(GameTicker.DefaultMap, map, new MapLoadOptions
+        var (_, gridId) = _mapLoader.LoadBlueprint(GameTicker.DefaultMap, map, new MapLoadOptions
         {
-            Offset = aabb.Center + MathF.Max(aabb.Height, aabb.Width)*2.5f
+            Offset = aabb.Center + MathF.Max(aabb.Height / 2f, aabb.Width / 2f) * 2.5f
```
Comment (PaulRitter) [inline Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs:106]: funnily enough the missing /2 was intended but we can move the shuttle closer too, idc. the idea was to give the crew a tad more of a buffer.

File: Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs [hunk 2]
```diff
@@ -113,19 +116,52 @@ private void OnPlayersSpawning(RulePlayerSpawningEvent ev)
...
-        var gear = _prototypeManager.Index<StartingGearPrototype>("SyndicateOperativeGearFull");
-        var spawnpos = new EntityCoordinates(_mapManager.GetGridEuid(grid.Value), new Vector2(1.5f, -4.5f));
+        var gridUid = _mapManager.GetGridEuid(gridId.Value);
+        // TODO: Loot table or something
+        var commanderGear = _prototypeManager.Index<StartingGearPrototype>("SyndicateCommanderGearFull");
+        var starterGear = _prototypeManager.Index<StartingGearPrototype>("SyndicateOperativeGearFull");
+
+        var spawns = new List<EntityCoordinates>();
+
+        // Forgive me for hardcoding prototypes
```
Comment (PaulRitter) [inline Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs:126]: reeeeeeee also hey spawnpoints, nice

**Review by PaulRitter — COMMENTED** [review]
p based

**Comment by PaulRitter** [comment]
haent review the map btw but i trust

File: Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs (hunk 1 above)
Comment (ike709) [reply Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs:106]: You could give the crew a buffer by disabling the shuttle for a few minutes for a "planning/equipping" phase

Comment (metalgearsloth) [reply Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs:106]: I know they have pinpointer but based on the average IQ I saw I feared them getting lost.

**Pull request merged**